### DISCUSSION TITLE
1st Announcement 🥳

### DISCUSSION THREAD

**Discussion by prrockzed** [prrockzed/chessed#11] [Announcements]
Hi people (contributors and players and everyone)
Hi all !!

This is the discussion section of the chessed platform..
You can put forward your thoughts, criticisms, ideas, contributions, bugs and basically everything, but it should strictly revolve around the **'chessed'** platform.

You are more than welcome to create your own discussion page according to what you want to discuss here. It can be an announcement, ideas, poll, QnA or anything in general.

**Comment by prrockzed** [comment]
I will soon be announcing on what are my future plans for the platform. Stay tuned!

**Comment by cloudblimp** [comment]
Hey Priyanshu! Can you tell me how the coordinates of the board are to be set?

**Comment by prrockzed** [comment]
Hi Shashank! Please create a new discussion to ask doubts. I will reply there.
Lets not talk about everything in one single discussion, it will get too messed up.
